packages/fxa-payments-server/Dockerfile

```diff
@@ -9,19 +9,45 @@ RUN npm ci
 COPY ["fxa-payments-server/.storybook", ".storybook/"]
 COPY ["fxa-payments-server/public", "public/"]
 COPY ["fxa-payments-server/src", "src/"]
+COPY ["fxa-shared", "../fxa-shared/"]
+WORKDIR /fxa-shared
+USER root
+RUN npm ci
+
+USER app
 COPY ["fxa-content-server", "../fxa-content-server/"]
 WORKDIR /fxa-content-server
 RUN npm ci
 WORKDIR /app
 ENV PUBLIC_URL /
 ENV INLINE_RUNTIME_CHUNK false
 RUN npm run build
-
-FROM node:12-stretch-slim
+FROM node:12-stretch
 RUN groupadd --gid 10001 app  && \
     useradd --uid 10001 --gid 10001 --home /app --create-home app
 USER app
 WORKDIR /app
 COPY --chown=app:app --from=node-builder /app .
 COPY --chown=app:app [ "fxa-payments-server/", "./" ]
+
+# Borrowing this stanza from fxa-content-server/Dockerfile-build
+# Except, to avoid installing redis, we're just copying over the two files
```

Comment:
I'm a bit confused by this. Are you trying to avoid installing the redis db, or the redis npm module? Copying over just a few files feels like a future footgun, I can imagine someone shouting in frustration "but why aren't my files there?! The payments server already uses fxa-shared, and the files aren't there!"

Also, is this comment out of date? It looks like 7 files are copied rather than 2.